platforms/winpack_dldt/2021.4/sysroot.config.py

```diff
@@ -0,0 +1,56 @@
+sysroot_bin_dir = prepare_dir(self.sysrootdir / 'bin')
+copytree(self.build_dir / 'install', self.sysrootdir / 'ngraph')
+#rm_one(self.sysrootdir / 'ngraph' / 'lib' / 'ngraph.dll')
+
+build_config = 'Release' if not self.config.build_debug else 'Debug'
+build_bin_dir = self.build_dir / 'bin' / 'intel64' / build_config
+
+def copy_bin(name):
+    global build_bin_dir, sysroot_bin_dir
+    copytree(build_bin_dir / name, sysroot_bin_dir / name)
+
+dll_suffix = 'd' if self.config.build_debug else ''
+def copy_dll(name):
+    global copy_bin, dll_suffix
+    copy_bin(name + dll_suffix + '.dll')
+    copy_bin(name + dll_suffix + '.pdb')
+
+copy_bin('cache.json')
+copy_dll('clDNNPlugin')
+copy_dll('HeteroPlugin')
+copy_dll('inference_engine')
+copy_dll('inference_engine_ir_reader')
+copy_dll('inference_engine_ir_v7_reader')
+copy_dll('inference_engine_legacy')
+copy_dll('inference_engine_transformations')  # runtime
+copy_dll('inference_engine_lp_transformations')  # runtime
+copy_dll('inference_engine_preproc')  # runtime
```

Comment:
@dkurt @ilya-lavrenov Hi, do you know if we need to add `inference_engine_preproc.dll` for inferencing on IE through OpenCV DNN module?

Reply:
From what I see in dnn code - we don't trigger IE preprocessing. 

As a dummy check you can run dnn tests and if they don't load preprocessing dynamically, it's not used in opencv. Can be checked via `strace`

Reply:
Yes, dnn does not utilize OpenVINO preprocessing

Reply:
@ilya-lavrenov @dkurt  Thank you!
Removed this .dll from the package (opencv_test_dnn tests pass).